{"links": {"self": {"href": "data/repositories/osrf/subt/issues/225/comments/54256580.json"}, "html": {"href": "#!/osrf/subt/issues/225#comment-54256580"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/subt/issues/225.json"}}, "type": "issue", "id": 225, "repository": {"links": {"self": {"href": "data/repositories/osrf/subt.json"}, "html": {"href": "#!/osrf/subt"}, "avatar": {"href": "data/bytebucket.org/ravatar/{3c95f6ad-c304-407c-b838-09597d836552}ts=2272898"}}, "type": "repository", "name": "subt", "full_name": "osrf/subt", "uuid": "{3c95f6ad-c304-407c-b838-09597d836552}"}, "title": "Local/AWS docker simulation does not always start"}, "content": {"raw": "As long as you\u2019re sending velocity commands to the correct topic, the robot should move.  Is there any way your Python script failed to pass a blocking statement before sending commands?  Do you have a printout to see when it begins sending commands to the robot?", "markup": "markdown", "html": "<p>As long as you\u2019re sending velocity commands to the correct topic, the robot should move.  Is there any way your Python script failed to pass a blocking statement before sending commands?  Do you have a printout to see when it begins sending commands to the robot?</p>", "type": "rendered"}, "created_on": "2019-10-08T14:18:12.132140+00:00", "user": {"display_name": "Arthur Schang", "uuid": "{1403972a-9b64-442b-a69c-d7e9557e1d87}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B1403972a-9b64-442b-a69c-d7e9557e1d87%7D"}, "html": {"href": "https://bitbucket.org/%7B1403972a-9b64-442b-a69c-d7e9557e1d87%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/initials/AS-0.png"}}, "nickname": "Arthur Schang", "type": "user", "account_id": "557058:14b7fd50-d65a-47a8-825c-fa89681d07bd"}, "updated_on": null, "type": "issue_comment", "id": 54256580}